packages/lesswrong/lib/types/typeUtil.ts

```diff
@@ -4,6 +4,15 @@ export {}
 
 declare global {
 
+// The version of the typescript lib we use is missing several of the valid
+// constructors for `Error`, so we add them here
+interface ErrorConstructor {
```

Comment:
I've found myself wanting to use the `cause` parameter a few times recently but we currently need to use `@ts-ignore` - this fixes that